src/main/scala/io/iohk/ethereum/network/PeerManagerActor.scala

```diff
@@ -152,14 +167,17 @@ class PeerManagerActor(
 
     if (nodesToConnect.nonEmpty) {
       log.debug("Trying to connect to {} nodes", nodesToConnect.size)
-      nodesToConnect.foreach(n => self ! ConnectToPeer(n.toUri))
+      nodesToConnect.foreach(n => {
+        triedNodes.put(n.id, n)
```

Comment:
maybe we should mark node as tried also when we connect to it via random tries - https://github.com/input-output-hk/mantis/pull/903/files#diff-00a5e82132cd8fdaa408d88c385d12965f491c99a68c704c8a7db90a481dbff4R138 ?

Reply:
yes, makes total sense. Was thinking about it also, but forgot to add